src/modules/site-v2/templates/strain/landing_page.html

```diff
@@ -2,12 +2,60 @@
 
 {% block content %}
 
+<style>
+    h2, h3 {
+        text-align: center;
+        color: black;
+        font-weight: bold;
+    }
+    .card a {
+        text-decoration: none;
+        color: black
+    }
+    .container {
+        display: flex;
+        justify-content: center;
+    }
+    .card {
+        text-align: center;
+        width: 25rem;
+        margin: 2rem
+    }
+    @media only screen and (max-width: 770px) {
+        .container {
+            flex-direction: column;
+        }
+        .card {
+            width:auto
+        }
+    }
+</style>
 
-<div>
-    <h2>Request Strains Landing Page</h2>
+<div class="container-fluid">
+    <h1>Request Strains</h1>
+    <h2>Please choose your affiliation:</h2>
```

Comment:
Should use the following instead of custom CSS: 

`<h2 class="text-dark text-center"> ... </h2>`